src/components/aria-live/README.md

```diff
@@ -0,0 +1,37 @@
+# MdAriaLive
```

Comment:
I believe this does make the most sense as a component, since you can inject it both ways.